cmd/signature-v4.go

```diff
@@ -35,6 +35,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/AchievementNetwork/stringset"
```

Comment:
This stringset dependency is not necessary @ebozduman we already have `github.com/minio/minio-go/v7/pkg/stringset`

Reply:
Done